tools/mmp/driver.cs

```diff
@@ -138,6 +135,27 @@ public static partial class Driver {
 		public static bool IsUnifiedMobile { get { return TargetFramework == TargetFramework.Xamarin_Mac_2_0_Mobile; } }
 		public static bool LinkProhibitedFrameworks { get; private set; }
 
+		static string mono_prefix;
+		static string MonoPrefix {
+			get {
+				if (mono_prefix == null) {
+					mono_prefix = Environment.GetEnvironmentVariable ("MONO_PREFIX");
```

Comment:
why not `Environment.GetEnvironmentVariable ("MONO_PREFIX") ?? "/Library/Frameworks/Mono.framework/Versions/Current"` ?

Reply:
GetEnvironmentVariable could possibly return string.Empty.